.zenodo.json

```diff
@@ -116,6 +116,11 @@
       "affiliation": "Neuroscience Program, University of Iowa",
       "orcid": "0000-0002-3339-4857"
     },
+    {
+      "name": "Heinsfeld, Anibal S.",
```

Comment:
Sorry I wasn't clear. You should insert your name right before Russ. I'll merge as soon as that gets fixed.

Thanks!

Reply:
Yeah, I was in doubt about it, definitely should've asked.

Done!